packages/Webkul/Admin/src/Resources/views/components/products/dynamic-attribute-fields.blade.php

```diff
@@ -137,6 +140,32 @@ class="text-xs text-gray-600 dark:text-gray-300 font-medium cursor-pointer selec
                     width='210px'
                 />
                 @break
+            @case('gallery')
+                @php
+                $savedImages = !empty($value) ? array_map(function ($image, $index) {
+                    return [
+                        'id'    => uniqid(),
+                        'url'   => Storage::url($image),
+                        'value' => $image,
+                    ];
+                },  (array)$value, array_keys((array)$value)) : [];
+                @endphp
+
+                @if (! empty($value))
+                    <!-- Empty value sent when value is deleted need to send empty value for this field -->
+                    <input type="hidden" name="{{ $fieldName }}" value="" multiple>
```

Comment:
Is there a Usage of multiple attribute in hidden input.

Reply:
Removed